From d4136941971acecce86e16b5224668e266d2468e Mon Sep 17 00:00:00 2001
From: Roman Stratiienko <roman.stratiienko@globallogic.com>
Date: Mon, 10 Dec 2018 17:47:33 +0200
Subject: [PATCH] Replace bootlin/libdrm-sun4i dependency with intel/libdrm

Since libdrm-2.4.95 "Arm AFBC modifiers" was introduced

Replace allwinner specific DRM_FORMAT_MOD_ALLWINNER_MB32_TILED with
DRM_FORMAT_MOD_ARM_AFBC(1)

Signed-off-by: Roman Stratiienko <roman.stratiienko@globallogic.com>
---
 src/video.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/video.c b/src/video.c
index b8a75a0..956d403 100644
--- a/src/video.c
+++ b/src/video.c
@@ -51,7 +51,7 @@ static struct video_format formats[] = {
 		.v4l2_buffers_count	= 1,
 		.v4l2_mplane		= false,
 		.drm_format		= DRM_FORMAT_NV12,
-		.drm_modifier		= DRM_FORMAT_MOD_ALLWINNER_MB32_TILED,
+		.drm_modifier		= DRM_FORMAT_MOD_ARM_AFBC(1),
 		.planes_count		= 2,
 		.bpp			= 16
 	},
-- 
2.17.1

